components/composer/src/components/HTMLEditor.svelte

```diff
@@ -12,17 +13,20 @@
 
   let container: HTMLDivElement;
   let toolbar: ToolbarItem[] = defaultActions;
-  let htmlUpdated = false;
 
   $: if (focus_body_onload && container) {
     container.focus();
   }
 
-  $: if (html) {
-    if (!htmlUpdated) {
-      htmlUpdated = true;
+  onMount(() => {
+    // If html exist, means composer is loaded with a previous message,
+    // wrap it with border-left css to visually differentiate the current message
+    if (html) {
       html = `<div><br/><br/><div style="border-left: 3px solid #dfe1e8; padding-left: 1rem;">${html}</div></div>`;
```

Comment:
We want to wrap it only once on load if html exists. If we wrap it in the markup, it will reactively update everytime the html is updated. When a user wants to create a new message, and they're not passing in any html, and then someone starts typing, it will add the css.

Reply:
I understand why this was implemented this way, my concern is just about the amount code smell this introduces. Building out markup in strings can be a recipe for all sorts of strange bugs in the future. I don't have all the context for the requirements here, but IMO this whole feature seems like something that the parent app should be handling instead of the composer. I won't hold up the review process, but keep in mind that it will be difficult to build on top of this solution in the future.

Reply:
I see what you mean. Agreed that handling this in the parent app is probably a better solution and this is more so spoon feeding the user. If this ends up opening a new can of worms, we shall rethink the solution. Thanks for pointing it out, Oliver.